app/hub/Views/HomeView/HomeView.jsx

```diff
@@ -34,6 +34,9 @@ const HomeView = (props) => {
 		isPlus,
 	} = props;
 	const accountHref = `https://account.${globals.GHOSTERY_DOMAIN}.com`;
+  const headerInfoText = (globals.BROWSER_INFO && globals.BROWSER_INFO.name && globals.BROWSER_INFO.name === 'firefox')
```

Comment:
Can we shorten this line to:

`(globals.BROWSER_INFO && globals.BROWSER_INFO.name === 'firefox')`

since the `BROWSER_INFO.name` won't equal `'firefox'` if it's undefined, anyway?

Reply:
done